documentation/assemblies/configuring/assembly-storage.adoc

```diff
@@ -13,11 +13,15 @@ The supported storage types are:
 * Ephemeral (Recommended for development only)
 * Persistent
 * JBOD (Kafka only; not available for ZooKeeper)
+* Tiered storage (Technology preview)
 
-To configure storage, you specify `storage` properties in the custom resource of the component. 
+To configure non-tiered storage, you specify `storage` properties in the custom resource of the component. 
 The storage type is set using the `storage.type` property.
 
-When using node pools, you can specify storage configuration unique to each node pool used in the cluster.
+Support for tiered storage is being introduced as a technology preview.
+To configure tiered storage, you specify `tieredStorage` properties.
+
+When using node pools, you can specify storage configuration unique to each node pool used in a Kafka cluster.
```

Comment:
This should be probably before you start talking abotu tiered storage. And here you can say that tiered storage is configured only at the cluster level in the Kafka CR.